File: use-cases/customer-growth-job-service/e2e/deploy/golang-sqlx-service/pkg/entity/customer_growth_job_entity.go
package entity

type CustomerGrowthJobEntity struct {
	ID          int64  `db:"id"`
	Region      string `db:"region"`
	TenantID    string `db:"tenant_id"`
	WorkspaceID string `db:"workspace_id"`
	ProjectID   string `db:"project_id"`
	JobID       string `db:"job_id"`
	DisplayName string `db:"display_name"`
	Status      string `db:"status"`
	OwnerUserID string `db:"owner_user_id"`
}

type CustomerGrowthJobQueryCriteria struct {
	WorkspaceID string
	ProjectID   string
	Status      string
	OwnerUserID string
}

File: use-cases/customer-growth-job-service/e2e/deploy/springboot-jdbc-service/src/main/java/com/authguard/usecases/entity/CustomerGrowthJobEntity.java
package com.authguard.usecases.entity;

public record CustomerGrowthJobEntity(
    Long id,
    String region,
    String tenantId,
    String workspaceId,
    String projectId,
    String jobId,
    String displayName,
    String status,
    String ownerUserId) {}
